nyc_taxi/anaconda-project.yml

```diff
@@ -15,21 +15,20 @@ examples_config:
 user_fields: [examples_config]
 
 channels:
-- pyviz/label/dev
 - defaults
 
 packages: &pkgs
 - python=3.11.8
-- notebook>=7.0.8
+- notebook<7
```

Comment:
There's in fact no need to pin `notebook<7` for projects that don't have a notebook deployment. Or in other words, only those that have this in their config must have the pin:

```yaml
  deployments:
    - command: "notebook"
```

Reply:
OK.